Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -177,13 +177,15 @@ void SVGFormattingContext::run(AvailableSpace const& available_space)
     // NOTE: SVG doesn't have a "formatting context" in the spec, but this is the most
     //       obvious way to drive SVG layout in our engine at the moment.
 
-    bool is_standalone_svg_document = !this->context_box()               // the [context] box
```

Comment:
Same with `LibWeb: Protect against nullptr for active_document() `

Fixup into the first commit.